Tiltfile

```diff
@@ -0,0 +1,28 @@
+###################
+# VECTOR TILTFILE #
+###################
+
+load('ext://helm_resource', 'helm_resource', 'helm_repo')
+
+docker_build(
+    ref='timberio/vector',
+    context='.',
+    build_args={'RUST_VERSION': '1.58'},
```

Comment:
I'm not super happy with this but I didn't see a way to pull it from the toolchain file, probably is a way though